deploy/operator.yaml

```diff
@@ -20,7 +20,8 @@ spec:
           ports:
           - containerPort: 8383
             name: metrics
-          args: ["start"]
+          args:
+            - start
```

Comment:
Changed this, as it's easier to add/remove `--log-level=debug` with this syntax.